### PR TITLE
Started obelisk app deployment guide

### PR DESCRIPTION
**Pull request by ingenieroariel**
Adds a guide to start and deploy obelisk apps from a custom branch. Useful for release QA procedures.

This was tested against the qa branch `c7f02b` on NixOS/OSX/iOS/Android.

cc @jmininger

### CODE REVIEW

File: guides/app-deploy.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+First, let's open a terminal, navigate to the path of your choice and define a variable called `WORKDIR` to help us avoid running commands in the wrong folder.
+
+~~~
+export WORKDIR=$(pwd)
+~~~
+
+Then let's get obelisk from Github using the qa branch:
+
+~~~
+git clone https://github.com/obsidiansystems/obelisk -b qa "$WORKDIR/obelisk"
```
Comment (iraiyan) [inline guides/app-deploy.md]: QA branch does not exist

File: guides/app-deploy.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+```nix
+  services.openssh.enable = true;
+  services.openssh.permitRootLogin = "yes";
+```
+
+and then Ctrl+O and Ctrl+X.
+
+Now run `sudo nixos-rebuild switch` and then set the root password to anything you want using `sudo passwd root`.
+
+With that you will be able to SSH into that machine from a terminal, using the username root and the IP address you found out during the previous step:
```
Comment (iraiyan) [inline guides/app-deploy.md]: Clarify that users will run `root@192.168.5.185` from their host terminal

File: guides/app-deploy.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+environment.systemPackages = with pkgs; [
+     git
+  ];
+```
+
+and then do another: `nixos-rebuild switch`.
+
+As the remote `root` user:
+```bash
+git init --bare ~/myapp.git
```
Comment (iraiyan) [inline guides/app-deploy.md]: Running on the VM terminal, display "Initialized empty Git repository in /root/myapp.git/"

File: guides/apple-developer-setup.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+6. Now let's add certificate for our newly created Apple ID:
+   Open Keychain Access on the Mac:
+   ![Screen Shot 2019-12-20 at 4.46.34 PM](./assets/Screen%20Shot%202019-12-20%20at%204.46.34%20PM.png)
+
+And after clicking on Certificates and System, use the Certificate Assistant to Request a Certificate From a Certificate Authority and Save it to a file:
+
+
+
+![Screen Shot 2019-12-20 at 4.47.42 PM](./assets/Screen%20Shot%202019-12-20%20at%204.47.42%20PM.png)
```
Comment (iraiyan) [inline guides/apple-developer-setup.md]: Should the Certificate download in "Login" or "System"?

**Comment by 3noch** [comment]
Closed in favor of #672 which does not include the apple dev setup guide.

**Pull request closed without merging**